### PR TITLE
Release 3.0.1-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description
Looking for the PR for the upcoming second beta of the v3.0.1 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "3.0.1-beta2": [
+      "[Fixed] Stream output from merge-tree - #14485",
```
Comment (sergiou87) [inline changelog.json]: Not sure if this is the best wording (any suggestions @niik?), but definitely related to what the users experienced.
```suggestion
      "[Fixed] Fix crash determining mergeability of a branch with many conflicts - #14485",
```

**Review by sergiou87 — CHANGES_REQUESTED** [review]

File: changelog.json (hunk 1 above)
Comment (niik) [reply changelog.json]: Yeah, that's a great improvement. How about something like

```suggestion
      "[Fixed] Prevent crash when encountering a large number of conflicts while checking for ability to merge branch - #14485",
```

A bit wordy but uses the terminology of desktop and .com rather than my (possibly) made up word "mergeability"

Comment (sergiou87) [reply changelog.json]: Sold! I'd say we can deploy this one already

**Review by sergiou87 — APPROVED** [review]
